src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(Color color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getArea() {
+        return side * side;
+    }
+
+    public String draw() {
+        return "Figure: Square, Area: "
+                + getArea()
+                + " units, side: "
+                + side
+                + " units, "
+                + " Figure color: "
+                + getColor();
```

Comment:
The draw() method can be part of an interface that this class implements, such as Drawable, if the draw() and getArea() methods should not be in the same interface due to conceptual differences.